src/app/forms/draft-court-order-conviction/draft-court-order-conviction.pdf.component.js

```diff
@@ -87,23 +87,10 @@ export default class DraftCourtOrderConviction_Pdf extends React.Component {
           <PositionedString dataKey="case.judgeName" left="54.1%" top="55.1%" />
 
           <PositionedCheckmark
-            dataKey="case.resolutionType"
-            left="12.1%"
-            top="68.8%"
-            shouldShow={data.case.resolutionType === "hearing"}
-          />
-
-          <PositionedCheckmark
-            dataKey="case.resolutionType"
+            debugKey="resolutionType"
             left="12.1%"
             top="65.7%"
-            shouldShow={data.case.resolutionType === "pleadings"}
-          />
-
-          <PositionedString
-            dataKey="case.resolutionDate"
-            left="35%"
-            top="68.9%"
+            shouldShow={true}
```

Comment:
We're just always showing a `hearing` checkmark? No option to change it to `pleadings`?

Reply:
Based off what tucker comments this is the one that should show.